sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/ADIS16448Protocol.scala

```diff
@@ -0,0 +1,92 @@
+package com.lynbrookrobotics.potassium.sensors.imu
+
+import java.nio.ByteBuffer
+
+import com.lynbrookrobotics.potassium.sensors.SPI
+import squants.motion.{AngularVelocity, DegreesPerSecond}
+
+/**
+  * Implements the IMU protocol for the robot.
+  *
+  * https://cdn-shop.adafruit.com/datasheets/L3GD20H.pdf <- look here for more info
+  */
+class ADIS16448Protocol(spi: SPI) {
+
+  private object Registers {
```

Comment:
This, and other objects holding constants, should be moved into a companion object.